system-configuration/src/network_configuration.rs

```diff
@@ -0,0 +1,427 @@
+use core_foundation::array::CFArray;
+use core_foundation::base::{CFType, TCFType};
+use core_foundation::base::kCFAllocatorDefault;
+use core_foundation::dictionary::CFDictionary;
+use core_foundation::string::{CFString, CFStringRef};
+
+use dynamic_store::SCDynamicStoreBuilder;
+pub use system_configuration_sys::network_configuration::*;
+use system_configuration_sys::preferences::SCPreferencesCreate;
+
+use std::{fmt, ptr};
+use std::net::IpAddr;
+
+#[derive(Debug)]
+pub struct SCNetworkInterfaceMTU {
+    cur: u32,
```

Comment:
I'm not a big fan of abbreviations where they are not very standard ones or not really needed to reduce bloat. Here I would argue we would benefit from calling it `current` to be more user friendly. In other words, we can call it `cur` when interacting with the `sys` layer, but `current` in the abstraction.